racecar_beacon/racecar_beacon/ros_monitor.py

```diff
@@ -1,66 +1,71 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import rclpy
+from rclpy.executors import ExternalShutdownException
 from rclpy.node import Node
-import socket
-import threading
-from struct import *
+from .lab_poll_pos import yaw_from_quaternion
```

Comment:
A would prefer to define a utils file at the place of using code from a lab
```suggestion
from racecar_beacon.utils import yaw_from_quaternion
```